src/mon/MonClient.h

```diff
@@ -302,13 +304,15 @@ class MonClient : public Dispatcher,
 
   LogClient *log_client;
   bool more_log_pending;
+  bool dispatch_queue_throttle=false;
```

Comment:
```suggestion
  std::atomic_bool dispatch_queue_throttle=false;
```
?

Reply:
Yes, I already have the `atomic` in my testing code. But it didn't resolve the reading issue in get_health_metrics. However, the new timestamp method would eliminate this variable it seems. Will work on the timestamp recording.